packages/preview-middleware/src/base/flex.ts

```diff
@@ -76,9 +77,17 @@ export async function readChanges(project: ReaderCollection, logger: Logger): Pr
  * @returns object with success flag and optional message
  */
 export function writeChange(
-    data: object & { fileName?: string; fileType?: string },
+    data: object & {
+        fileName: string;
+        fileType: string;
+        changeType: string;
```

Comment:
We should probably be less strict with `changeType` as well and have it as optional property and later default to some `unknown` type perhaps, so we don't get `undefined` value in the telemetry data.

Reply:
done